sdk/python/kfp_tekton/compiler/_data_passing_rewriter.py

```diff
@@ -634,13 +634,23 @@ def append_taskrun_params(task_name_append: str):
                         dst = '$(results.%s.path)' % sanitize_k8s_name(result['name'])
                         if artifact_name == result['name'] and src != dst:
                             add_copy_results_artifacts_step = True
-                            script += (
-                                    'ARTIFACT_SIZE=`wc -c %s | awk \'{print $1}\'`\n' % src +
-                                    'TOTAL_SIZE=$( expr $TOTAL_SIZE + $ARTIFACT_SIZE)\n' +
-                                    'touch ' + dst + '\n' +  # create an empty file by default.
-                                    'if [[ $TOTAL_SIZE -lt 3072 ]]; then\n' +
-                                    '  if ! awk "/[^[:print:]]/{f=1} END{exit !f}" %s; then\n' % src +
-                                    '    cp ' + src + ' ' + dst + '\n' +
+                            copy_results_artifact_step['script'] += (
```

Comment:
We want to move away from tekton script to tekton container spec so that it can create the task pod with fewer init-containers.
```suggestion
                            script += (
```